### DISCUSSION TITLE
Configuring People Search using People entity from Microsoft Search data source

### DISCUSSION THREAD

**Discussion by rajeshb4** [microsoft-search/pnp-modern-search#1679] [Q&A]
Hi Team,

I am trying to use the newly added People entity from Microsoft Search data source to configure People Search for our organization. Unfortunately all I get in the results are small person icon without any profile details. Can someone please let me know how to use the  people entity?
Note: I made sure I checked the beta endpoint toggle.
![image](https://user-images.githubusercontent.com/19842672/147690851-d8686a83-0a11-43e0-9138-960bb2a17e1a.png)
![image](https://user-images.githubusercontent.com/19842672/147690909-32dc3816-a2ad-4865-902d-4066c580047a.png)
![image](https://user-images.githubusercontent.com/19842672/147691048-947d651a-e2a0-4582-a24b-f6981b1b1ea0.png)

**Comment by wobba** [comment]
People is beta and I get an error testing on my tenant for any people query as passing in `fields` break the request. I suggest using the SharePoint API for people still until the API stabilize somewhat.

**Reply by rajeshb4** [reply]
@wobba Thanks for your reply. I was using SharePoint API previously but that brought disabled users from the tenant. In order to avoid that I thought I could use the new People entity. Although I could remove disabled users by modifying the query template but there is no way I could remove them from org chart of other users.

**Reply by FranckyC** [reply]
People entity type does not work well actually :(. For instance you can't get all people at once with the wildcard character as it does not seems to support KQL. For now you should rely on SharePoint search. For now what you can do is pretty much this https://docs.microsoft.com/en-us/graph/people-example#perform-a-fuzzy-search
